pymc3/distributions/multivariate.py

```diff
@@ -1292,3 +1295,221 @@ def logp(self, value):
         n = self.n
         norm = - 0.5 * m * n * pm.floatX(np.log(2 * np.pi))
         return norm - 0.5*trquaddist - m*half_collogdet - n*half_rowlogdet
+
+
+class KroneckerNormal(Continuous):
```

Comment:
This supersedes MatrixNormal right?  If so, it might be good to change MatrixNormal to just be an alias for KroneckerNormal, to reduce code duplication.  If it happens it should probably be a separate PR.

Reply:
Ah yes that is true.   I'm perfectly OK with leaving things as is.